closure/testing/test/lenient_test.js

```diff
@@ -0,0 +1,15 @@
+goog.setTestOnly();
+goog.require('goog.testing.asserts');
+goog.require('goog.testing.jsunit');
+goog.require('goog.testing.testSuite');
+
+goog.scope(function() {
+    const Lenient = goog.require('rules_closure.Lenient');
+    
+    goog.testing.testSuite({
+      testMe: function() {
+        assertEquals(Lenient(), 123);
+      }
+    });
+    
+});  // goog.scope
```

Comment:
could you follow the simpler style:

testSuite({
  mytest() { ... }
});

Also could you document the part the you expect an error but will pass since it is lenient?

Reply:
If i don't use a `goog.scope`, the compiler complains with:
```
closure/testing/test/lenient_test.js:5: ERROR - Closure primitive method goog.require may not be aliased  outside a module (ES module, CommonJS module, or goog.module)
const Lenient = goog.require('rules_closure.Lenient');
```

Reply:
Okay, I've changed it to a goog.module.  FWIW, `entry_points` needed to be set, which isn't necessary with a goog.scope.